crawl-ref/source/describe-god.cc

```diff
@@ -548,6 +553,106 @@ static formatted_string _god_wrath_description(god_type which_god)
     return desc;
 }
 
+static formatted_string _beogh_extra_description()
+{
```

Comment:
would be nice to add something like "last seen on" to these descriptions, for anyone not currently in los. Maybe too involved for the initial iteration, though.